Source/SIMPLib/CoreFilters/Testing/Cxx/MultiThresholdObjectsTest.cpp

```diff
@@ -47,13 +46,32 @@
 #include "SIMPLib/Plugin/SIMPLibPluginLoader.h"
 #include "SIMPLib/SIMPLib.h"
 #include "SIMPLib/Testing/UnitTestSupport.hpp"
+#include "SIMPLib/DataContainers/DataContainerArray.h"
+#include "SIMPLib/DataContainers/DataContainer.h"
 
 class MultiThresholdObjectsTest
 {
 public:
   MultiThresholdObjectsTest() = default;
   virtual ~MultiThresholdObjectsTest() = default;
-  SIMPL_TYPE_MACRO(MultiThresholdObjectsTest)
+  /**
+   * @brief Returns the name of the class for MultiThresholdObjectsTest
+   */
```

Comment:
And another duplicate description.

Reply:
This seems to be the case for much of the Testing/Cxx directory.  Consider this a recurring comment rather than making note of all the rest of the test classes.